### PR TITLE
Added BeanFactory sample in spring-core

### PR DESCRIPTION
**Pull request by sbmaggarwal**

### CODE REVIEW

File: spring-core/pom.xml [hunk 1]
```diff
@@ -50,6 +50,11 @@
             <artifactId>javax.inject</artifactId>
             <version>1</version>
         </dependency>
+        <dependency>
```
Comment (mklew) [inline spring-core/pom.xml]: There is junit version 4.11 already in this pom. I believe you don't need yet another junit

File: spring-core/src/main/java/com/baeldung/beanfactory/BeanFactoryWithClassPathResource.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import org.springframework.core.io.Resource;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class BeanFactoryWithClassPathResource {
+
+    @Test
+    public void createBeanFactory() {
+
```
Comment (mklew) [inline spring-core/src/main/java/com/baeldung/beanfactory/BeanFactoryWithClassPathResource.java]: please remove this blank line

File: spring-core/src/main/java/com/baeldung/beanfactory/BeanFactoryWithClassPathResource.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class BeanFactoryWithClassPathResource {
+
+    @Test
+    public void createBeanFactory() {
```
Comment (mklew) [inline spring-core/src/main/java/com/baeldung/beanfactory/BeanFactoryWithClassPathResource.java]: test name should be more meaningful, take a look at the name in the article

**Comment by sbmaggarwal** [comment]
@mklew 

I updated the dependency.

**Pull request merged**